doc/source/user_guide/indexing.rst

```diff
@@ -675,8 +675,7 @@ Current behavior
    Passing list-likes to .loc with any non-matching elements will raise
    KeyError in the future, you can use .reindex() as an alternative.
 
-   See the documentation here:
-   https://pandas.pydata.org/pandas-docs/stable/indexing.html#deprecate-loc-reindex-listlike
+   See the documentation :ref:`here <indexing.deprecate_loc_reindex_listlike>`
```

Comment:
I think this is also an output from pandas, not an actual link that we want to change, isn't it?

Reply:
Yes this looks like a similar case, @jorisvandenbossche can you confirm?

Reply:
Yes